roles/metrics/README.md

```diff
@@ -0,0 +1,126 @@
+
+## About
+
+This readme contains the steps for deploying [`Prometheus`](https://prometheus.io/) monitoring application, [`Loki`](https://grafana.com/oss/loki/) to aggregation logs, [`Promtail`] ('https://grafana.com/docs/loki/latest/clients/promtail/') agent which ships the contents of local logs to a private Grafana Loki instance or Grafana Cloud and [`Grafana`](https://grafana.com/grafana/) which is used for visualize and analyze metrics and logs generated by Prometheus and Loki.
+
+## Disclaimer
+
+With a focus on improving the service, aiming at greater maintainability and programmability of the solution. We started a refactoring of the service using [`Helm`](https://helm.sh/), initially we are porting grafana and then we will porte all the rest of the observability solution.
+
+As the inclusion of services (Prometheus, Loki, Promtail and Grafana) is under development, some services are currently not being monitored. The following describes the services already implemented.
+
+## Node Exporter
+
+[`Node Exporter`](https://github.com/prometheus/node_exporter) is used for get host metrics from the host itself. In other words, the metrics obtained by Prometheus are directly related to hardware and operating system.
+
+Available metrics:
+
+* Service uptime;
+* Number of CPU cores;
+* Total amount of RAM;
+* Total amount of containers;
+* Disk usage percentage;
+* RAM usage percentage;
+* Total amount of SWAP;
+* Network traffic (Sent and Received);
+* CPU usage percentage;
+* Total, free and avaliable RAM;
+* Amount of RAM for I/O processes;
+* Used space of available disks.
+
+## cAdvisor
+
+[`cAdvisor`](https://github.com/google/cadvisor) is an exporter developed by Google that provides metrics regarding resource usage and performance characteristics of running containers.
+
+Available metrics:
+
+* Current status of the container: ``Running``, ``Stopped`` and ``Paused``;
+* Network traffic received by container;
+* Network traffic sent by container;
+* Percentage of RAM usage per container;
+* Percentage of CPU usage per container;
+* RSS Memory Usage per Container;
+* RAM consumption table per container;
+* Available RAM for each conatainer;
+* RAM limit established for each container.
+
+## For Kubernetes
+
+For [`Kubernetes`](https://kubernetes.io/) we have several specific dashboards for monitoring the cluster, nodes, pods and others services, as mentioned below:
+
+* k8s resources namespaces workloads;
+* Dojot 100K;
+* Locust;
+* Vmq cluster;
+* Vmq Nodes;
+* k8s resources cluster;
+* k8s resources namespaces pods;
+* k8s resources namespaces pods;
+* k8s resources workloads;
+* k8s nodes;
+* k8s persistent volumes;
+* k8s pods;
+* k8s statefulsets;
+* k8s usemethod cluster;
+* k8s usemethod node.
+
+
+## Using the service
+
+Since we are using Docker, it is natural that we will use its metrics. To set the Docker daemon as a Prometheus target, you need to specify the metrics-address in ``/etc/docker/daemon.json``.
```

Comment:
Vamos usar o docker junto com o K8S ?

Não seria possível incluir essas configurações no playbook ?

Reply:
Esse trecho é básicamente oara você conseguir obter dados do docker, usar o docker, não é necessário. No entato, se for preciso, a gente consegue obter as métricas. E a respeito do playbook, de fato, pode ser uma possibilidade. No decorrer, das inclusões, nas configurações necessárias eu irei fazer dessa forma.